src/components/modal/Modal.driver.new.ts

```diff
@@ -7,8 +7,10 @@ export const ModalDriver = (props: ComponentProps) => {
   const driver = useComponentDriver<ModalProps>(props);
   const overlayDriver = ButtonDriver({renderTree, testID: `${testID}.TouchableOverlay`});
 
+  const containerDriver = useComponentDriver({renderTree, testID: `${testID}.modalContainer`});
+
   const isVisible = () => {
-    return !!driver.getProps().visible;
+    return containerDriver.exists();
```

Comment:
This change could have been in a different PR, but I like this change to a view existing instead of a prop check